src/components/Staking/StakingLaunchpadWidget.tsx

```diff
@@ -1,12 +1,11 @@
 import { useState } from "react"
-import { Box, chakra, Flex } from "@chakra-ui/react"
+import { chakra, Box, Flex, Select } from "@chakra-ui/react"
 import { FaTools } from "react-icons/fa"
+import { useTranslation } from "next-i18next"
 
 import { ButtonLink } from "@/components/Buttons"
 import Text from "@/components/OldText"
-import { StyledSelect as Select } from "@/components/SharedStyledComponents"
```

Comment:
Fixing this. We should keep using this Select component instead of the chakra one